package.json

```diff
@@ -33,6 +33,7 @@
     "@clark/eslint-config-ember": "^1.1.1",
     "@clark/eslint-config-ember-typescript": "^1.1.1",
     "@clark/eslint-config-node": "^1.1.1",
+    "@ember-decorators/component": "^6.0.0",
```

Comment:
Oh sorry, this one has to go into `dependencies`.

Reply:
🤦‍♂ My bad.

Reply:
I also noticed I'm still using our company's npm registry. Will fix that too.